plugins/system/shortcut/src/Extension/Shortcut.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  System.shortcut
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Plugin\System\Shortcut\Extension;
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Event\GenericEvent;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Uri\Uri;
+use Joomla\Event\Event;
+use Joomla\Event\SubscriberInterface;
+use Joomla\Registry\Registry;
+
+/**
+ * Shortcut plugin to add accessible keyboard shortcuts to the administrator templates.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+final class Shortcut extends CMSPlugin implements SubscriberInterface
+{
+	/**
+	 * Load the language file on instantiation.
+	 *
+	 * @var    boolean
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected $autoloadLanguage = true;
+
+	/**
+	 * Returns an array of events this subscriber will listen to.
+	 *
+	 * The array keys are event names and the value can be:
+	 *
+	 *  - The method name to call (priority defaults to 0)
+	 *  - An array composed of the method name to call and the priority
+	 *
+	 * For instance:
+	 *
+	 *  * array('eventName' => 'methodName')
+	 *  * array('eventName' => array('methodName', $priority))
+	 *
+	 * @return  array
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public static function getSubscribedEvents(): array
+	{
+		return [
+			'onBeforeCompileHead' => 'initialize',
+			'onLoadShortcuts'     => 'addShortcuts',
+		];
+	}
+
+	/**
+	 * Add the javascript for the shortcuts
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function initialize()
+	{
+		if (!$this->getApplication()->isClient('administrator'))
+		{
+			return;
+		}
+
+		$context = $this->getApplication()->input->get('option') . '.' . $this->getApplication()->input->get('view');
+
+		$shortcuts = [];
+
+		$event = new GenericEvent(
+			'onLoadShortcuts',
+			[
+				'context'   => $context,
+				'shortcuts' => $shortcuts,
+			]
+		);
+
+		$this->getDispatcher()->dispatch('onLoadShortcuts', $event);
+
+		$shortcuts = $event->getArgument('shortcuts');
+
+		Text::script('PLG_SYSTEM_SHORTCUT_OVERVIEW_HINT');
+		Text::script('PLG_SYSTEM_SHORTCUT_OVERVIEW_TITLE');
+		Text::script('PLG_SYSTEM_SHORTCUT_OVERVIEW_DESC');
+		Text::script('JCLOSE');
+
+		$document = $this->getApplication()->getDocument();
+		$wa       = $document->getWebAssetManager();
+		$wa->useScript('bootstrap.modal');
+		$wa->registerAndUseScript('script', 'plg_system_shortcut/shortcut.min.js', ['dependencies' => ['hotkeys.js']]);
+
+		$plugin = PluginHelper::getPlugin('system', 'shortcut');
+
+		$timeout = (new Registry($plugin->params))->get('timeout', 2000);
```

Comment:
```suggestion
		$timeout = $this->params->get('timeout', 2000);
```
Should that also be possible here?

Reply:
Indeed!